salt/cloud/clouds/ec2.py

```diff
@@ -2856,6 +2862,53 @@ def create(vm_=None, call=None):
     __utils__['cloud.cache_node'](node, __active_provider_name__, __opts__)
     ret.update(node)
 
+    # Add any block device tags specified
+    ex_blockdevicetags = {}
+    blockdevicemappings_holder = block_device_mappings(vm_)
+    if blockdevicemappings_holder:
+        for _bd in blockdevicemappings_holder:
+            if 'tag' in _bd:
+                ex_blockdevicetags[_bd['DeviceName']] = _bd['tag']
+
+    block_device_volume_id_map = {}
+
+    if ex_blockdevicetags:
+        for k, v in six.iteritems(ret['blockDeviceMapping']):
```

Comment:
Can you switch these variables (and others below) to use 3 or more letters? That would be nice and match our [Style Guide](https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#variables) rules.

Reply:
sure thing